TgeeT/AutoMart
# #165901192 add update price for seller and buyer

#### What does this PR do?
Add functionality to update prices
#### Description of Task to be completed?
Have an interface for seller and buyers to update their initial price and orders respectively
#### How should this be manually tested?
After cloning this repository, open the index.html file in your browser you should see an update order button for an initially placed order, this is true for the sellers page as well
#### Any background context you want to provide?
This is just a mare interface with no working functionalities
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/165901192
#### update price modal
![image](https://user-images.githubusercontent.com/45405036/57901777-132e1180-786f-11e9-8324-ac0355ba973c.png)

#### Buyer option to trigger update
![image](https://user-images.githubusercontent.com/45405036/57901830-483a6400-786f-11e9-8704-b7d8842882ee.png)

#### seller option to trigger update
![image](https://user-images.githubusercontent.com/45405036/57901860-643e0580-786f-11e9-81bf-2894f98a9111.png)
